src/world/park.h

```diff
@@ -44,7 +44,7 @@ enum {
 	PARK_FLAGS_LOCK_REAL_NAMES_OPTION = (1 << 15),
 	PARK_FLAGS_NO_MONEY_SCENARIO = (1 << 17),  // equivalent to PARK_FLAGS_NO_MONEY, but used in scenario editor
 	PARK_FLAGS_18 = (1 << 18),
-	PARK_FLAGS_SIX_FLAGS_DEPRECATED = (1 << 19)
+	PARK_FLAGS_SIX_FLAGS_DEPRECATED = (1 << 19) // Not used anymore
```

Comment:
It should not be, as it was used for:
1) Six Flags advertising
2) Hiding fatalities (from crashes, drowning, etc.) in Six Flags parks.